controllers/logging/log.go

```diff
@@ -42,41 +42,30 @@ func newLogger(config *depresolver.Config) *loggerFactory {
 // In such case returns default logger
 func (l *loggerFactory) get() zerolog.Logger {
 	var logger zerolog.Logger
-	var dt = time.RFC822Z
 	if l.log.Format == depresolver.NoFormat {
-		l.log.Format = depresolver.JSONFormat
+		l.log.Format = depresolver.SimpleFormat
 	}
 	if l.log.Level == zerolog.NoLevel {
 		l.log.Level = zerolog.InfoLevel
 	}
 	// We can retrieve stack in case of pkg/errors
 	zerolog.ErrorStackMarshaler = pkgerrors.MarshalStack
 	zerolog.SetGlobalLevel(l.log.Level)
-	if l.log.Level <= zerolog.DebugLevel {
-		dt = "15:04:05"
```

Comment:
Do we want `RFC822Z` for Debug?   
```
09:06:19 DBG controllers/logging/log.go:80 > Logger settings: [simple, debug]
or
02 Jan 06 15:04 -0700 DBG controllers/logging/log.go:80 > Logger settings: [simple, debug]
```
Debugging is mostly manual, for better readability I wanted to put shorter time format. @ytsarev , @k0da  Thoughts ?

Reply:
actually it will be different
```
09:06:19 DBG controllers/logging/log.go:80 > Logger settings: [simple, debug]
vs
2021-03-23T15:04:05Z07:00 DBG controllers/logging/log.go:80 > Logger settings: [simple, debug]
```
as I changed datetime settings to `time. RFC3339` for all log formats.